src/Whoops/Resources/css/whoops.base.css

```diff
@@ -157,6 +157,14 @@ header {
     }
 
     .frame-index {
+      float: left;
```

Comment:
floats can always be avoided and replaced with a better `inline-block` or `display: flex`.

In this case, perhaps simply setting `.frame-index, .frame-method-info { display: inline-block }` solves the problem better.

 - [ ] Try inline-blocks instead of floats